internal/consts/consts.go

```diff
@@ -454,6 +454,9 @@ const (
 	EnvVarRadiusPassword = "RADIUS_PASSWORD"
 	// EnvVarTokenFilename for the TokenFile auth login.
 	EnvVarTokenFilename = "TERRAFORM_VAULT_TOKEN_FILENAME"
+
+	// EnvVarVaultConfigPath to override where the Vault configuration is.
+	EnvVarVaultConfigPath = "VAULT_CONFIG_PATH"
```

Comment:
Do we need to document this env variable?

Reply:
It's a constant that is only used to override the config in tests and is not user-facing, so we should be good on that front! I also added a doc string to the constant here mentioning that it is only for tests, and that the TFVP does not rely on the constant to read the actual config for Vault, to make things clearer :)